src/APIInterfaces/BackendAPI/Requests/CourseRequests.ts

```diff
@@ -585,3 +586,13 @@ export const deleteAttachments = async ({
         throw new BackendAPIError(e);
     }
 };
+
+export const getAllContentForVersion = async ({
+    gradeInstanceId
+}: {gradeInstanceId: number}): Promise<AxiosResponse<any>> => {
```

Comment:
This is bad, we should try not to make a habbit of it (using any and typing inline)
Also even if we are rushing it should be `BackendAPIResponser<any>`